### PR TITLE
Add Beluga AMCL usage guide

### PR DESCRIPTION
**Pull request by hidmic**
### Proposed changes

This patch adds a Beluga AMCL usage guide, pending since #346.

#### Type of change

- [ ] 🐛 Bugfix (change which fixes an issue)
- [ ] 🚀 Feature (change which adds functionality)
- [x] 📚 Documentation (change which fixes or extends documentation)

### Checklist

- [ ] Lint and unit tests (if any) pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] All commits have been signed for [DCO](https://developercertificate.org/)

### CODE REVIEW

**Review by nahueespinosa — APPROVED** [review]
LGTM

File: docs/guides/using-beluga-amcl.md [hunk 1]
```diff
@@ -1,3 +1,153 @@
...
+Remember to source your ROS 2 environment to use it:
+
+```bash
+source /opt/ros/${ROS_DISTRO}/setup.bash
+```
+:::
+
+## Mapping the environment
+
+Before the robot can localize using Beluga AMCL, we need a map of the environment. Here's how you can create one:
```
Comment (glpuga) [inline docs/guides/using-beluga-amcl.md:45]: Some parts of the text use `’` and others `'`, like here in `Here's`.

File: docs/guides/using-beluga-amcl.md [hunk 2]
```diff
@@ -1,3 +1,153 @@
...
+    <arg name="map_path" description="Path to you map.yaml file." />
+
+    <!-- Load map -->
+    <node pkg="nav2_map_server" exec="map_server" name="map_server" output="screen">
+        <param name="yaml_filename" value="$(var map_path)" />
+    </node>
+
+    <!-- Start AMCL -->
+    <node pkg="beluga" exec="amcl" name="amcl" output="screen">
+        <param name="use_sim_time" value="false" />
```
Comment (glpuga) [inline docs/guides/using-beluga-amcl.md:90]: Maybe this one needs not be included at all.

File: docs/guides/using-beluga-amcl.md [hunk 3]
```diff
@@ -1,3 +1,153 @@
...
+
+1. **Beluga AMCL fails to converge**
+
+   - **Symptom:** the particle cloud doesn't cluster around the robot's true pose.
+   - **Possible causes:**
+     - Incorrect LiDAR or odometry data.
+     - Map doesn't match the actual environment.
+   - **Crosschecks:**
+     - Verify the `/scan` topic data is correct using `ros2 topic echo /scan`.
+     - Verify the `odom` to `base_link` transforms are correct using `ros2 run tf2_ros tf2_echo odom base_link`.
```
Comment (glpuga) [inline docs/guides/using-beluga-amcl.md:142]: Also check lidar to base_link.

File: docs/guides/using-beluga-amcl.md [hunk 4]
```diff
@@ -1,3 +1,153 @@
...
+     - Check the map matches the actual environment.
+
+2. **Robot's estimated pose is inaccurate**
+
+   - **Symptom:** the robot's displayed pose doesn't match its actual pose.
+   - **Possible causes:**
+     - Odometry drift.
+     - Incorrect transformations between frames.
```
Comment (glpuga) [inline docs/guides/using-beluga-amcl.md:150]: Blurry/distorted map too.

File: docs/guides/using-beluga-amcl.md [hunk 5]
```diff
@@ -1,3 +1,153 @@
...
+    <!-- Load map -->
+    <node pkg="nav2_map_server" exec="map_server" name="map_server" output="screen">
+        <param name="yaml_filename" value="$(var map_path)" />
+    </node>
+
+    <!-- Start AMCL -->
+    <node pkg="beluga" exec="amcl" name="amcl" output="screen">
+        <param name="use_sim_time" value="false" />
+        <param name="min_particles" value="500" />
+        <param name="max_particles" value="2000" />
```
Comment (glpuga) [inline docs/guides/using-beluga-amcl.md:92]: Most of these are defaults, can we omit them?

File: docs/guides/using-beluga-amcl.md (hunk 1 above)
Comment (hidmic) [reply docs/guides/using-beluga-amcl.md:45]: Good catch! Fixed in https://github.com/Ekumen-OS/beluga/pull/447.

File: docs/guides/using-beluga-amcl.md (hunk 2 above)
Comment (hidmic) [reply docs/guides/using-beluga-amcl.md:90]: Removed in https://github.com/Ekumen-OS/beluga/pull/447

File: docs/guides/using-beluga-amcl.md (hunk 3 above)
Comment (hidmic) [reply docs/guides/using-beluga-amcl.md:142]: True that. Done in https://github.com/Ekumen-OS/beluga/pull/447.

File: docs/guides/using-beluga-amcl.md (hunk 4 above)
Comment (hidmic) [reply docs/guides/using-beluga-amcl.md:150]: Good point. Added it in https://github.com/Ekumen-OS/beluga/pull/447.

File: docs/guides/using-beluga-amcl.md (hunk 5 above)
Comment (hidmic) [reply docs/guides/using-beluga-amcl.md:92]: Yes we can omit them all. Done in https://github.com/Ekumen-OS/beluga/pull/447.

**Pull request merged**